components/Main/Workouts/Workouts.js

```diff
@@ -1,29 +1,67 @@
 import { StyledWorkouts, StyledWorkoutsHeader } from "./StyledWorkouts";
 import Image from "next/image";
-import Link from "next/link";
+import InputWorkout from "./InputWorkout/InputWorkout";
+import { useState } from "react";
+import React from "react";
+
+const Workouts = ({ workouts, onAddWorkout, onDelete }) => {
+  const [showInput, setShowInput] = useState(false);
+
+  function toggleShow() {
+    setShowInput(true);
```

Comment:
```suggestion
    setShowInput(!showInput);
```
dann gehts auch wieder zu, wenn man nochmal klickt :)

Reply:
No geht leider nicht :)

Reply:
mysteriös. dann lass es so, alles prima :)